frontend/ui/widget/opdsbrowser.lua

```diff
@@ -58,7 +58,35 @@ local OPDSBrowser = Menu:extend{
 }
 
 function OPDSBrowser:init()
-    self.item_table = self:genItemTableFromRoot(self.opds_servers)
+    local servers = G_reader_settings:readSetting("opds_servers")
+    if not servers then -- If there are no saved servers, add some defaults
+        servers = {
+          {
+            title = "Project Gutenberg",
+            url =
+            "http://m.gutenberg.org/ebooks.opds/?format=opds",
```

Comment:
nitpick, please keep it consistent with other blocks, i.e. no put `url = ` and the string into one line.